src/modules/client/components/customers/ProfileFollowUp.vue

```diff
@@ -188,12 +187,10 @@ export default {
   computed: {
     auxiliaryAvatar () {
       let auxiliaryPicture;
-      if (this.customer.referent.picture) {
-        auxiliaryPicture = this.customer.referent.picture;
-      }
+      if (get(this.customer, 'referent.picture')) auxiliaryPicture = this.customer.referent.picture;
```

Comment:
Tu peux mettre `const auxiliaryPicture = get(this.customer, 'referent.picture', null)` non ?

Reply:
tres juste